src/components/EntityPage/Participant/ParticipantSummary.js

```diff
@@ -157,6 +157,16 @@ function specimenSummaryTableData(specimen) {
 
 const ParticipantSummary = ({ participant }) => {
   const specimens = get(participant, 'biospecimens.hits.edges', []);
+  const hasFile = get(participant, 'files.hits.edges', [] ).length === 0 ? false : true
```

Comment:
Another way to do this could be:
``const hasFile = !!get(participant, 'files.hits.edges', []).length``
Try it out sometime. :)